webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -6931,4 +6251,699 @@ const AdminDefinition: AdminDefinitionType = {
     },
 };
 
+export const ldapWizardAdminDefinition: AdminDefinitionSubSectionSchema = {
+    id: 'LdapSettings',
+    name: defineMessage({id: 'admin.authentication.ldap.wizard', defaultMessage: 'AD/LDAP Wizard'}),
+    sections: [{
+        key: 'admin.authentication.ldap.connection',
+        title: 'Connection',
+        subtitle: 'Connection and security level to your AD/LDAP server.',
```

Comment:
Also, these need to be translatable. That can be done in the followup PR though to avoid making this conflict with the next one PR more than we need to

Reply:
Ah, good point, I'll make a TODO note. Thanks!